common/scripts/access-control/flexible-server/Grant-FlexibleServerDBAccess.ps1

```diff
@@ -0,0 +1,112 @@
+<#
+.SYNOPSIS
+Grant access to postgres flexible server database for service (tier-3) managed identity.
+
+.DESCRIPTION
+Grant access to postgres flexible server database for service (tier-3) managed identity.
+
+.EXAMPLE
+.\Grant-FlexibleServerDBAccess.ps1 
+#>
+
+Set-StrictMode -Version 3.0
+
+[string]$PostgresHost = $env:POSTGRES_HOST
+[string]$PostgresDatabase = $env:POSTGRES_DATABASE
+[string]$ServiceMIName = $env:SERVICE_MI_NAME
+[string]$TeamMIName = $env:TEAM_MI_NAME
+[string]$TeamMIClientId = $env:AZURE_CLIENT_ID
+[string]$TeamMITenantId = $env:AZURE_TENANT_ID
+[string]$TeamMISubscriptionId = $env:TEAM_MI_SUBSCRIPTION_ID
+[string]$TeamMIFederatedTokenFile = $env:AZURE_FEDERATED_TOKEN_FILE
+[string]$SubscriptionName = $env:SUBSCRIPTION_NAME
+[string]$WorkingDirectory = $PWD
+
+[string]$functionName = $MyInvocation.MyCommand
+[DateTime]$startTime = [DateTime]::UtcNow
+[int]$exitCode = -1
+[bool]$setHostExitCode = (Test-Path -Path ENV:TF_BUILD) -and ($ENV:TF_BUILD -eq "true")
+[bool]$enableDebug = (Test-Path -Path ENV:SYSTEM_DEBUG) -and ($ENV:SYSTEM_DEBUG -eq "true")
+
+Set-Variable -Name ErrorActionPreference -Value Continue -scope global
+Set-Variable -Name VerbosePreference -Value Continue -Scope global
+
+if ($enableDebug) {
+    Set-Variable -Name DebugPreference -Value Continue -Scope global
+    Set-Variable -Name InformationPreference -Value Continue -Scope global
+}
+
+Write-Host "${functionName} started at $($startTime.ToString('u'))"
+Write-Debug "${functionName}:PostgresHost:$PostgresHost"
+Write-Debug "${functionName}:PostgresDatabase:$PostgresDatabase"
+Write-Debug "${functionName}:ServiceMIName:$ServiceMIName"
+Write-Debug "${functionName}:TeamMIName:$TeamMIName"
+Write-Debug "${functionName}:TeamMIClientId=$TeamMIClientId"
+Write-Debug "${functionName}:TeamMIFederatedTokenFile=$TeamMIFederatedTokenFile"
```

Comment:
Is it going to write any sensitive information?

Reply:
@ppotturi done